1-mandatory/1-writers.js

```diff
@@ -62,7 +64,12 @@ Exercise 2:
 */
 
```

Comment:
A few points about this one:
1. Same point as above about using `let` or `const`
2. Not a big deal, but when you see `author.alive === false` in your code, you can write it like `!author.alive`. Can you think of why that works?
3. Now for extra practice - can you re-write this using array methods (maybe you can use `filter` first, and then `forEach`)?